admin_guide/quota.adoc

```diff
@@ -6,140 +6,169 @@
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 == Overview
 
-A `ResourceQuota` defines constraints that limit aggregrate resource consumption
-per project.  It can limit the quantity of objects that can be created in a project
-by type, as well as the total amount of link:compute_resources.html[compute resources]
-that may be consumed by resources in that project.
+A resource quota, defined by a `*ResourceQuota*` object, provides constraints
+that limit aggregate resource consumption per project. It can limit the quantity
+of objects that can be created in a project by type, as well as the total amount
+of compute resources that may be consumed by resources in that project.
 
-== Resources managed by quota
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+[NOTE]
+====
+See the link:../dev_guide/compute_resources.html[Developer Guide] for more on
+compute resources.
+====
+endif::[]
 
-The following describes the set of resources that may be managed by a `ResourceQuota`.
+[[resources-managed-by-quota]]
+== Resources Managed by Quota
+
+The following describes the set of resources that may be managed by a resource
+quota.
 
-.Resources managed by quota
 [cols="3a,8a",options="header"]
 |===
 
 |Resource Name |Description
 
-|`configmaps`
-|Total number of configmaps
-
-|`cpu`
-|Across all pods in a non-terminal state, the sum of cpu requests cannot exceed this value.
+|`*cpu*`
+|Across all pods in a non-terminal state, the sum of CPU requests cannot exceed
+this value.
 
-|`memory`
-|Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value.
+|`*memory*`
+|Across all pods in a non-terminal state, the sum of memory requests cannot
+exceed this value.
 
-|`requests.cpu`
-|Across all pods in a non-terminal state, the sum of cpu requests cannot exceed this value.
+|`*requests.cpu*`
+|Across all pods in a non-terminal state, the sum of CPU requests cannot exceed
+this value.
 
-|`requests.memory`
-|Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value.
+|`*requests.memory*`
+|Across all pods in a non-terminal state, the sum of memory requests cannot
+exceed this value.
 
-|`limits.cpu`
-|Across all pods in a non-terminal state, the sum of cpu limits cannot exceed this value.
+|`*limits.cpu*`
+|Across all pods in a non-terminal state, the sum of CPU limits cannot exceed
+this value.
 
-|`limits.memory`
-|Across all pods in a non-terminal state, the sum of memory limits cannot exceed this value.
+|`*limits.memory*`
+|Across all pods in a non-terminal state, the sum of memory limits cannot exceed
+this value.
 
-|`configmaps`
-|The total number of config maps that can exist in the project.
+ifdef::openshift-origin,atomic-registry[]
```

Comment:
I am not sure when/how we remember to change this when next version drops, but I assume that is something you guys track?